zdf_2_csv_without_zivid.py

```diff
@@ -0,0 +1,46 @@
+"""
+Convert ZDF point cloud to CSV format without Zivid Software.
+"""
+
+import numpy as np
+from netCDF4 import Dataset
+
+
+def _main():
+
+    # The Zivid3D.zdf file has to be in the same folder as this sample script.
+    filename_zdf = "Zivid3D.zdf"
+    filename_csv = "Zivid3D.csv"
+
+    print(f"Reading {filename_zdf} point cloud")
+    data = Dataset(filename_zdf)
```

Comment:
Can you use a context manager here instead?

Reply:
Done